treehouses/remote
# settings: add led patterns (fixes #899)

Fixes #899
**How to test:**
Go to **Settings**, **LED Pattern** and choose a pattern. Then click on **Test Bluetooth** on the home screen to test the pattern. You can also look at the raspberry pi LEDs to see if they are blinking correspondingly.

_Note_
Some led patterns might not exactly match its visualization on the app yet. While working on this, I found some minor discrepancies between some led commands and their description. 
For example, in led valentine, the red led turned off for 1 second instead of the correct 5 seconds.
So some pull requests may still need to be merged in cli for some visualizations to match their led commands.
